tamr_toolbox/utils/operation.py

```diff
@@ -128,3 +128,31 @@ def wait(
             return operation
         operation = operation.poll()
     raise TimeoutError(f"Waiting for operation took longer than {timeout_seconds} seconds.")
+
+
+def monitor(
```

Comment:
Please add an example to the documentation for using this new function

Reply:
Added example snippets for wait and monitor